### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by MeirShpilraien**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,46 +12,47 @@ It supports reading [Wikipedia Abstract Data Dumps](https://dumps.wikimedia.org/
...
-## Some Results
+## Benchmarking
 
-Output of the benchmark, running on a full Abstracts dump of the English wikipedia, on 5 redis shards over a `c4.4xlarge` EC2 instance:
+### Benchmarking Redisearch
+The following benchmark is running on a full Abstracts dump of the English wikipedia, on 5 shards over a `c4.8xlarge` EC2 instance, performing two-word search queries with 32 clients:
+
+Steps:
+
+* Create a 5 shards redisearch cluster on redis entrprise with the following configuration:
```
Comment (mnunberg) [inline README.md]: shard

File: README.md [hunk 2]
```diff
@@ -12,46 +12,47 @@ It supports reading [Wikipedia Abstract Data Dumps](https://dumps.wikimedia.org/
...
+```
+PARTITION AUTO MAXDOCTABLESIZE 10000000
+```
+
+* Populate Data:
+```
+```
+
+* Running benchmark:
```
Comment (mnunberg) [inline README.md]: Run the benchmark

File: README.md [hunk 3]
```diff
@@ -12,46 +12,47 @@ It supports reading [Wikipedia Abstract Data Dumps](https://dumps.wikimedia.org/
...
-autocomplete (2-3 letters) |32|16064.66|1.90
-autocomplete (2-3 letters) |64|17255.77|3.51
-autocomplete (2-3 letters) |128|17935.49|6.47
-
-
-## Benchmark output
+two-word search | 32 | 12547 | 8
+
+### Benchmarking Multi-tenant Redisearch
+The following benchmark test the amount of time it takes to create 50K indices with 500 docs in each index. total of 25 million documents. It uses the oss redisearch version on redis enterprise cluster.
```
Comment (mnunberg) [inline README.md]: tests

Comment (mnunberg) [inline README.md]: "The following benchmark tests the amount of time it takes to create 50,000 indexes with 500 docs in each index, for a total of 25M documents. It uses the enterprise version of RediSearch on Redis Enterprise Cluster

File: README.md [hunk 4]
```diff
@@ -12,46 +12,47 @@ It supports reading [Wikipedia Abstract Data Dumps](https://dumps.wikimedia.org/
...
+
+Steps:
+
+* Create a 20 shards redisearch cluster on redis entrprise.
```
Comment (mnunberg) [inline README.md]: shard

**Review by mnunberg — CHANGES_REQUESTED** [review]

**Comment by MeirShpilraien** [comment]
@mnunberg fixed.

**Review by mnunberg — APPROVED** [review]

**Pull request merged**